src/supermarket/app/views/pages/dashboard.html.erb

```diff
@@ -18,12 +18,14 @@
         <% @followed_cookbook_activity.each do |cookbook_version| %>
           <li>
             <div>
-              <%= link_to gravatar_for(cookbook_version.owner), cookbook_version.published_by %>
+              <%= link_to gravatar_for(cookbook_version.owner), cookbook_version.owner %>
               <%= t('cookbook.activity',
-                    maintainer: link_to(cookbook_version.published_by.name, cookbook_version.published_by),
+                    maintainer: link_to(cookbook_version.maintainer, cookbook_version.owner),
```

Comment:
Despite the parameter name to the I18n renderer being `maintainer`, here you'll want `cookbook_version.owner.name` instead of `cookbook_version.maintainer`.